scss/components/calendar.scss

```diff
@@ -108,14 +108,14 @@ $fd-calendar-table-width: $fd-calendar-table-item-width * 7 !default;
   }
 }
 
-
 .#{$block} {
-
   //BLOCK BASE *******************************************
   //set all BLOCK reset and baseline styles
   @include fd-reset;
   width: $fd-calendar-table-width;
+}
 
+.#{$block} .#{$block} {
```

Comment:
@xak ~ How do you feel about this? I've had some instances of the docs `customstyles.css` file causing some problems within components CSS because of specificity. The last couple times i've made changes outside of the context of the component, but this has me thinking that maybe components CSS should be more specific to prevent this from happening in unknown-more-complex environments with probably way more CSS. Even just adding the `.#{block}` reference before every style is just enough specificity to not allow it to be overwritten.

http://cl.ly/tCAr